com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md

```diff
@@ -0,0 +1,44 @@
+# Reflection probe usage
+
+## What is a probe
+
+Reflection probes and planar reflection probes provide indirect specular lighting.
+
+A probe capture its surroundings and will influence specular lighting in a specific volume.
+A reflection probe will capture all of its surroundings while a planar reflection probe will capture only one direction.
```

Comment:
A Reflection Probe captures all of its surroundings whereas a Planar Reflection Probe only captures one direction.

For more information on probes and how they work, see [Reflection Probes](Reflection-Probes-Intro.md).